apps/admin-ui/src/groups/components/GroupTree.tsx

```diff
@@ -136,14 +136,16 @@ export const GroupTree = ({
       id: group.id,
       name: (
         <Tooltip content={group.name}>
-          {(canViewDetails && (
+          {canViewDetails ? (
             <Link
               to={`/${realm}/groups/${joinPath(...groups.map((g) => g.id!))}`}
               onClick={() => setSubGroups(groups)}
             >
               {group.name}
             </Link>
-          )) || <span>{group.name}</span>}
+          ) : (
+            <span>{group.name}</span>
```

Comment:
A string is also a valid JSX element, so I think you can omit the `span` element here.
```suggestion
            group.name
```